### PR TITLE
Fix docs for pings

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: docs/user/pings/custom.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Applications can define metrics that are sent in custom pings. Unlike the built-
 
 Custom pings must be defined in a `pings.yaml` file, which is in the same directory alongside your app's `metrics.yaml` file.
 
+Ping names are limited to lowercase ASCII characters and hyphens and a maximum of 30 characters.
```
Comment (travis79) [inline docs/user/pings/custom.md]: Just a wording nit: 
```suggestion
Ping names are limited to lowercase letters and hyphens with a maximum of 30 characters.
```

File: docs/user/pings/custom.md [hunk 2]
```diff
@@ -58,6 +60,26 @@ override fun onCreate() {
 
 </div>
 
+<div data-lang="Swift" class="tab">
+
+In Swift, this object must be registered with Glean from your startup code
+(such as in your application's `application` method or a function called from that method).
```
Comment (travis79) [inline docs/user/pings/custom.md]: It probably wouldn't hurt to mention _which_ `application` method we recommend, as there are several.
```suggestion
(such as in your application's `UIApplicationDelegate` [`application(_:didFinishLaunchingWithOptions:)`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application) method or a function called from that method).
```

File: docs/user/pings/custom.md (hunk 1 above)
Comment (mdboom) [reply docs/user/pings/custom.md]: Why remove "ASCII"?  It's probably important to state that you can't use é, for example.

Comment (travis79) [reply docs/user/pings/custom.md]: ASCII `0X8A` == `é`

Comment (travis79) [reply docs/user/pings/custom.md]: Again, just a nit.  I'm fine if `ASCII` stays there, I know what is meant by it and most everyone else would too.

Comment (badboy) [reply docs/user/pings/custom.md]: Side stepped the whole issue by using the [same wording as in other places](https://docs.telemetry.mozilla.org/cookbooks/new_ping.html).

**Pull request merged**